types/index.d.ts

```diff
@@ -0,0 +1,80 @@
+// Type definitions for Chalk 2
+// Definitions by: Thomas Sauer <https://github.com/t-sauer>
+
+export = chalk;
+
+declare function chalk(...text: (string | number)[]): string;
+declare function chalk(text: TemplateStringsArray, ...placeholders: (string | number)[]): string;
```

Comment:
I know we allow any value here, but I think we should enforce a string as TS users want strict types. On line 6 and 7.